scripts/filter-noisy-assembler-warnings.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
```

Comment:
Shouldn't these all be `/usr/bin/env python3`?

Reply:
Yes, sorry. This patch mixed two objectives: py3 + absolute paths (for Debian). I should have kept the 2nd part out